proto/eth/wrapper/beacon_wrapper.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.eth.wrapper;
+
+import "proto/eth/v2/version.proto";
+import "proto/eth/v1/beacon_state.proto";
+import "proto/eth/v2/beacon_state.proto";
+
+message WrappedBeaconState {
```

Comment:
Could you add commentary on what this is intended for?

Reply:
oh sure. Added